core/libs/shared/src/core_ops.rs

```diff
@@ -43,7 +44,7 @@ where
             file_type,
             last_modified,
             last_modified_by,
-            shares: Vec::new(),
+            shares: Vec::new(), // todo
```

Comment:
shares use the sharer/sharee's usernames, so this will require us to add a public key -> username cache